locale/Malayalam.json

```diff
@@ -94,6 +94,13 @@
       "searchHint": "തിരയുക",
       "continueLabel": "തിരഞ്ഞെടുത്ത് തുടരുക"
     },
+    "hashMarkInformation": {
+      "CreatedBy": "ഉണ്ടാക്കിയത്",
+      "IssuedBy": "പുറപ്പെടുവിച്ചത്",
+      "CreatedAt": "സൃഷ്ടിച്ച തീയതി",
+      "HashLabel": "ഹാഷ്",
+      "Transaction": "ഇടപാട്"
+    },
```

Comment:
@sreepriyasree @smohan-dw you are ok with this?

Reply:
no changes required